src/gui/addnewtorrentdialog.cpp

```diff
@@ -34,6 +34,7 @@
 #include <QUrl>
 #include <QMenu>
 #include <QFileDialog>
+#include <QPushButton>
```

Comment:
Yes, it is needed. Otherwise:
```
../../src/gui/addnewtorrentdialog.cpp:148:45: error: invalid use of incomplete type ‘class QPushButton’
  ui->buttonBox->button(QDialogButtonBox::Ok)->setFocus();
                                             ^~
```